### PR TITLE
Refactor/namespace constants

### PR DESCRIPTION
**Pull request by Sharmyn28**
Here I changed the namespaces and keynames that we use in Datastore, instead of using the DataStore API with strings I changed to constants. Also, changed `android_settings` keynames to `general_settings`.

### CODE REVIEW

File: src/components/android-settings-container.js [hunk 1]
```diff
@@ -174,16 +165,15 @@ class AndroidSettingsContainer extends React.Component {
                                           numberSmsConfirmation: '',
                                           valuesTEI: '',
                                           encryptDB: encryptDB,
+                                          isUpdated: true,
+                                          loading: false,
```
Comment (amcgee) [inline src/components/android-settings-container.js]: This looks like a functional change, should it be in a different PR?

File: src/components/dataSet-settings.js [hunk 2]
```diff
@@ -370,15 +360,12 @@ class DataSetSettings extends React.Component {
...
-                                                const sumaryString =
-                                                    dataSet.periodDSDownload
                                                 const sumarySettings =
                                                     dataSet.periodDSDownload ===
                                                     undefined
                                                         ? undefined
-                                                        : this.arrangeWord(
-                                                              sumaryString
-                                                          )
+                                                        : dataSet.periodDSDownload
```
Comment (amcgee) [inline src/components/dataSet-settings.js]: This also looks like a functional change?

File: src/components/dataSet-settings.js [hunk 3]
```diff
@@ -234,11 +227,10 @@ class DataSetSettings extends React.Component {
                     .periodDSDBTrimming,
             }
 
-            const sumaryString = this.state.specificSetting.periodDSDownload
             const sumarySettings =
                 this.state.specificSetting.periodDSDownload === undefined
                     ? undefined
-                    : this.arrangeWord(sumaryString)
+                    : this.state.specificSetting.periodDSDownload
```
Comment (amcgee) [inline src/components/dataSet-settings.js]: This also looks like a functional change?

File: src/components/android-settings-container.js (hunk 1 above)
Comment (Sharmyn28) [reply src/components/android-settings-container.js]: I will remove it and create a new PR just for this change

File: src/components/dataSet-settings.js (hunk 2 above)
Comment (Sharmyn28) [reply src/components/dataSet-settings.js]: Indeed this is a functional change, the thing is that on my dataStore I was already using the input number so it sent me a render error, I think I'm also removing these lines in another PR, I'll keep the lines.

File: src/components/dataSet-settings.js (hunk 3 above)
Comment (Sharmyn28) [reply src/components/dataSet-settings.js]: I'll keep the lines

**Comment by Sharmyn28** [comment]
Hi @amcgee I just removed the functional settings
